lib/faraday_middleware/response/caching.rb

```diff
@@ -27,6 +27,9 @@ class Caching < Faraday::Middleware
     #           :ignore_params - String name or Array names of query params
     #                            that should be ignored when forming the cache
     #                            key (default: []).
+    #           :write_options - Hash of settings that should be passed as the third
```

Comment:
I did consider other names, but this felt like the best one, being the most explicit about where exactly these options go